tests/dataset_histograms/histogram_error_estimator_test.py

```diff
@@ -92,9 +92,28 @@ def test_sum_not_supported(self):
                 ValueError, "Only COUNT and PRIVACY_ID_COUNT are supported"):
             self._get_estimator(pipeline_dp.Metrics.SUM)
 
-    def test_get_ratio_dropped_l0(self):
+    @parameterized.parameters((0, 1), (1, 0.818181818181818),
```

Comment:
also, in test for estimate_rmse it is hard to verify that the expected numbers are correct, is it possible to write a short formula? we have iteration over partitions there, 10 of them, so maybe it will be not very small formula...

Reply:
Good point. Done

The numbers for L0 histogram is correct, since it's not about rows, but about (privacy_unit, partition) pairs.